driver/esp32/ili9XXX.py

```diff
@@ -576,3 +588,104 @@ def __init__(self,
             spihost, mhz, factor, hybrid, width, height, colormode, rot, invert, double_buffer, half_duplex, display_type=DISPLAY_TYPE_ILI9488,
             asynchronous=asynchronous, initialize=initialize)
 
+
+class gc9a01(ili9XXX):
+    # On the tested display the wirte direction and colormode appear to be
+    #reversed from how they are presented in the datasheet and so have been
+    #included here as new definitions
+
+    # GC9A01 displays handle rotation differently than ili9XXX displays
+    ROTATE = {
+        0: 0x40,
+        90: 0x20,
+        180: 0x80,
+        270: 0xE0
+    }
+
+    # GC9A01 displays handle colour mode differently than ili9XXX displays
+    COLOR_MODE = {
+        'RGB': 0x08,
+        'BGR': 0x00
+    }
+
+    def __init__(self,
+        miso=5, mosi=18, clk=19, cs=13, dc=12, rst=4, power=14, backlight=15, backlight_on=0, power_on=0,
+        spihost=esp.HSPI_HOST, mhz=60, factor=4, hybrid=False, width=240, height=240,
+        colormode='RGB', rot=180, invert=False, double_buffer=True, half_duplex=True,
+        asynchronous=False, initialize=True
+    ):
+
+        if lv.color_t.SIZE != 2:
+            raise RuntimeError('gc9a01 micropython driver requires defining LV_COLOR_DEPTH=16')
+        if hybrid:
+            raise RuntimeError('gc9a01 micropython driver does not currently support hybrid driver')
```

Comment:
Did you check, maybe the C function as is (or with very small change) can work for gc9a01 as well?  
What is the difference between gc9a01 and ili9341 flush functionality? I don't see you override "flush" so I suspect [the same C "flush" function](https://github.com/lvgl/lv_binding_micropython/blob/dadad31bdce92fc96ffac6b2805ac0d5d4b884eb/driver/esp32/espidf.c#L174) ili9341 is using could be used here as well.

Reply:
I have no knowledge of c and will be away for a few days so didn't have time to dig around.

I tested previously and got an error but having just tried now it seems to be working, I think  had changed something somewhere else and then reverted it, I will remove this.

Reply:
If this is working, could you change it to `hybrid=True`?
Performance improvement is significant.

Reply:
Sorry I missed that, yes I will change although it will have to be Monday as I am away for the weekend